docs/topics/Automated-Checks.md

```diff
@@ -0,0 +1,259 @@
+# Automated Checks
+
+The [](DataWorkerService.md) has numerous responsibilities, one of them being a data processing step known as automated
+checks. These checks are responsible for processing various portions of data depending on the current processing step
+for a particular piece of data.
+
+## Core Principals
+
+When designing this system, we did so with the following principles in mind:
+
+1. Human reviewers have authority over whether an entity is `Verified` or `Rejected`. As such, the system will never automatically assign these designations.
+2. The automatic application of the `PreRejected` status must be as accurate as possible, based on concrete rules.
+3. The process must be as transparent as possible. As such, the system tracks all changes to entities in the `audit` tables. Additionally, all entities have a `RejectionReason` enum which defines a combination of reasons why it was marked as rejected by either the system or human reviewer.
+4. Do not include entities which are not `Verified` in the tournament rating algorithm.
+    * This provides an added benefit of ensuring all generated statistics are valid. Even with manually submitted data, humans make mistakes. If unverified data is introduced into the rating & statistics systems, users will notice invalid statistics and the rating ladder itself will not be completely accurate.
```

Comment:
```suggestion
    * Even with manually submitted data, humans make mistakes. If unverified data is introduced, users may notice invalid stats or inaccurate rating calculations, and it may be easier for bad actors to influence the algorithm.
```